data/upgrades/title.json

```diff
@@ -320,7 +320,7 @@
       {
         "title": "Outrider",
         "type": "Title",
-        "ability": "While you perform an attack that is obstructed by an obstacle, the defender rolls 1 fewer defense die. After you fully execute a maneuver, if you moved through or overlapped an obstacle, you may remove 1 of your red or orange tokens.",
+        "ability": "While you perform a primary attack at range 3, roll 1 additional attack die. While you perform an attack that is obstructed by an obstacle, you may change one of the defender's [Evade] results to a [Focus] result.",
```

Comment:
the errata file currently has what looks like a typo ("by an obstacle" text missing). The physical card has it, and the SL cards have it, so seems like a misprint, seemed safer to put to this temporarily until we have a real answer from AMG

Reply:
amg updated the errata to match the card